modules/local/run_sequence_aligner.nf

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env nextflow
```

Comment:
```suggestion
```
This line is not needed to import a module in `main.nf` file

Reply:
Removed